src/external/rawspeed/data/cameras.xml

```diff
@@ -6249,6 +6249,17 @@
 		<Crop x="0" y="0" width="4599" height="3064"/>
 		<Sensor black="0" white="4095"/>
 	</Camera>
+	<Camera make="SONY" model="DSLR-A380">
+		<ID make="Sony" model="DSLR-A380">Sony DSLR-A380</ID>
```

Comment:
Yes they are the same. Output of tools/dngmeta.sh
`
DNG created by : Adobe DNG Converter 9.6 (Windows)
DNG Illuminant : 21 (should be 21)


$ nano -w src/external/adobe_coeff.c

{ "Sony DSLR-A380", { 6038,-1484,-579,-9145,16746,2512,-875,746,7218 } },
`